### ISSUE TITLE
FAGG Groundlayout

### ISSUE THREAD

**Issue by TimS-W** [VATSSA-Operations/Sector-Files#287]
**Description**

Describe the requirement for a new ground layout.

**To Do**

- [ ] KMZ Development
- [ ] KMZ Conversion to SCT Text File
- [ ] GNG Implementation of SCT Entries
- [ ] GNG Implementation of Text Labels
- [ ] GNG Implementation of Regions

**Ground Layout Development**

- [ ] 1 Runways
- [ ] 2 Taxiways
- [ ] 3 Aprons
- [ ] 4 Stopbars
- [ ] 5 Parking
- [ ] 6 Buildings
- [ ] 7 Labels
- [ ] 8 Stand Data

**Comment by TimS-W** [comment]
[FAGG.zip](https://github.com/VATSSA-Operations/Sector-Files/files/9158564/FAGG.zip)

**Comment by vCaptainMike** [comment]
Right, I'm gonna push this to 2209, mostly to have time to see the diffs to the current one. Might extract some elements from here though for 2208. Will follow up next week.
